<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Товары</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Товары </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="products_table"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addProduct">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editProduct">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Изменить товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort_button">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортировка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteProduct">
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить товар</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
